app/uploaders/portrait_uploader.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+class PortraitUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
I don't want to make the assumption that people are running this only on aws. Adding a note somewhere in the docs would be good though.